deploy/catalogsourceconfig.crd.yaml

```diff
@@ -2,12 +2,57 @@ apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   name: catalogsourceconfigs.marketplace.redhat.com
+  annotations:
+    displayName: Catalog Source Config
+    description: Represents an CatalogSourceConfig.
 spec:
   group: marketplace.redhat.com
   names:
     kind: CatalogSourceConfig
     listKind: CatalogSourceConfigList
     plural: catalogsourceconfigs
     singular: catalogsourceconfig
+    shortNames:
+    - csc
   scope: Namespaced
   version: v1alpha1
+  additionalPrinterColumns:
+  - name: TargetNamespace
+    type: string
+    description: The namespace where the operators will be enabled
+    JSONPath: .spec.targetNamespace
+  - name: Packages
+    type: string
+    description: List of operator(s) which will be enabled in the target namespace
```

Comment:
I think this should mention how to format the list. I recall it being comma separated without spaces, it should be specified here.